cogito/put_test.go

```diff
@@ -309,8 +309,8 @@ func TestPutterProcessInputDirNonExisting(t *testing.T) {
 func TestPutterSinks(t *testing.T) {
 	putter := cogito.NewPutter("dummy-API", hclog.NewNullLogger())
 
-	sinks := putter.Sinks()
-
+	sinks, err := putter.Sinks()
+	assert.Assert(t, err == nil)
```

Comment:
Or maybe this is better?: https://github.com/Pix4D/fly_helper/blob/15f99bdfd4f5462f4f7be321e8072c46cc346bac/pkg/pipelinesetter/pipelinesetter_test.go#L178

Reply:
> LGTM. I left some minor comments.
> 
> I don't like the commented test. So its a must that you merge the other PR into this one before, merging both to master.

Me neither, however I will merge the other PR here, this was already planned to not merge independently.

Reply:
> Or maybe this is better?: https://github.com/Pix4D/fly_helper/blob/15f99bdfd4f5462f4f7be321e8072c46cc346bac/pkg/pipelinesetter/pipelinesetter_test.go#L178

Yes, I'll do :)

Reply:
Fixed